problems/lib/problem2.rb

```diff
@@ -0,0 +1,26 @@
+class Problem2
+  def get_largest_two_numbers(array)
+    raise "Array must have at least two elements." if array.size < 2
+
+    if (array[1] > array[0])
+      max1 = array[1]
+      max2 = array[0]
+    else
+      max1 = array[0]
+      max2 = array[1]
+    end

```

Comment:
I can see two possibilities at the moment:

1. Use parallel assignment like this:
`max1, max2 = array[0] > array[1] ? array[0..1] : array[0..1].reverse`
although this approach seems not to be recommended, after I scanned the [Ruby coding style](https://github.com/bbatsov/ruby-style-guide) you provided on your comment.
"Avoid the use of parallel assignment for defining variables. Parallel assignment is allowed when it is the return of a method call, used with the splat operator, or when used to swap variable assignment. Parallel assignment is less readable than separate assignment."

2. Extract code in a separate method.
`max1, max2 = get_largest_first_two_numbers(array)`

```
def get_largest_first_two_numbers(array)
  array[0] > array[1] ? array[0..1] : array[0..1].reverse
end
```